src/App.tsx

```diff
@@ -21,42 +41,22 @@ export const App: React.FC = () => {
               </div>
 
               <div className="block" data-cy="MainContent">
-                <p data-cy="NoSelectedUser">
-                  No user selected
-                </p>
-
-                <Loader />
+                {!selectedUser && (
```

Comment:
It's better to keep conditional rendering out of the return statement for cleaner code. Consider assigning conditional rendering to a variable and then using that variable in the return statement.